website/content/docs/upgrading/upgrade-to-1.18.x.mdx

```diff
@@ -36,3 +38,7 @@ WARNING! The following warnings were returned from Vault:
 ```
 
 </CodeBlockConfig>
+
+
+The `current_billing_period` toggle for `/sys/internal/counters/activity` is also deprecated, as this will be set
```

Comment:
Sorry for the delay and not bringing this up on Enterprise, but can you add a warning output like above? It helps users get more context about the warning if they encounter it.